python/housinginsights/ingestion/LoadData.py

```diff
@@ -312,6 +313,7 @@ def _create_zone_facts_table(self):
         # create empty zone_facts table
         metadata = MetaData(bind=self.engine)
         zone_facts = Table('zone_facts', metadata,
+                           Column('zone_type', String),
```

Comment:
Hey @jkwening sorry for the confusion. After some back and forth, where we landed was: 
a) it seemed like too much trouble for not enough benefit to rewrite the ingestion functions to use the ORM - what's there now is working and there's not much different behavior that the ORM would provide
b) Jason found the Automap feature, which allows us to reflect the current database structure into ORM objects without directly mapping them
c) We would need to custom-build the display_name, display_text etc. stuff for the front end in some way anyways even if we refactored into the orm

So - we get some benefits in the API by using Automap and Flask-Restless can use it for some autogenerated endpoints. But, we'll keep the ingestion as is. AS I said fine to use the ORM for this table if you want but we should still put the table into meta.json for the display_text and display_name purposes anyways, so probably better to just do it wholesale. Either approach is fine though.